libs/booking/feature-seat-listing/src/lib/booking-feature-seat-listing.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'aap-listing-info',
```

Comment:
How about `aap-seat-listing` since it's the `*SeatListingComponent`?

Reply:
I believe that what @LayZeeDK suggested is the right selector here.

Reply:
Oh yeah, you are right